docs/source/quickstart.mdx

```diff
@@ -83,23 +239,50 @@ outputs = pipeline(
+  - [question answering](https://github.com/huggingface/optimum-habana/tree/main/examples/question-answering)
+  - [sentence transformers training](https://github.com/huggingface/optimum-habana/tree/main/examples/sentence-transformers-training)
+  - [summarization](https://github.com/huggingface/optimum-habana/tree/main/examples/summarization)
+  - [table detection](https://github.com/huggingface/optimum-habana/tree/main/examples/table-detection)
+  - [text classification](https://github.com/huggingface/optimum-habana/tree/main/examples/text-classification)
+  - [text feature extraction](https://github.com/huggingface/optimum-habana/tree/main/examples/text-feature-extraction)
+  - [text generation](https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation)
+  - [translation](https://github.com/huggingface/optimum-habana/tree/main/examples/translation)
+  - [trl](https://github.com/huggingface/optimum-habana/tree/main/examples/trl)
+
+- **Audio**
+  - [audio classification](https://github.com/huggingface/optimum-habana/tree/main/examples/audio-classification)
   - [speech recognition](https://github.com/huggingface/optimum-habana/tree/main/examples/speech-recognition)
-- Text and images
-  - [text-to-image generation](https://github.com/huggingface/optimum-habana/tree/main/examples/stable-diffusion),
-  - [contrastive image-text training](https://github.com/huggingface/optimum-habana/tree/main/examples/contrastive-image-text).
+  - [text to speech](https://github.com/huggingface/optimum-habana/tree/main/examples/text-to-speech)
+
+- **Images**
+  - [object detection](https://github.com/huggingface/optimum-habana/tree/main/examples/object-detection)
+  - [object segementation](https://github.com/huggingface/optimum-habana/tree/main/examples/object-segementation)
+  - [image classification](https://github.com/huggingface/optimum-habana/tree/main/examples/image-classification)
+  - [image to text](https://github.com/huggingface/optimum-habana/tree/main/examples/image-to-text)
+  - [contrastive image text](https://github.com/huggingface/optimum-habana/tree/main/examples/contrastive-image-text)
+  - [stable diffusion](https://github.com/huggingface/optimum-habana/tree/main/examples/stable-diffusion)
+  - [visual question answering](https://github.com/huggingface/optimum-habana/tree/main/examples/visual-question-answering)
+  - [zero-shot object detection](https://github.com/huggingface/optimum-habana/tree/main/examples/zero-shot-object-detection)
+
+- **Video**
+  - [stable-video-diffusion](https://github.com/huggingface/optimum-habana/tree/main/examples/stable-diffusion)
+  - [video-classification](https://github.com/huggingface/optimum-habana/tree/main/examples/video-classification)
+
+To learn more about how to migrate your own model, check out [Model Conversion](https://huggingface.co/docs/optimum/habana/usage_guides/model_conversion).
```

Comment:
We need to update this link since we changed the name "Model conversion"

Reply:
Fixed